engine/wasmrunnable.go

```diff
@@ -1,44 +1,46 @@
-package rwasm
+package engine
 
 import (
 	"encoding/json"
 
+	"github.com/suborbital/reactr/api"
+	"github.com/suborbital/reactr/engine/moduleref"
+	"github.com/suborbital/reactr/engine/runtime"
 	"github.com/suborbital/reactr/request"
-	"github.com/suborbital/reactr/rt"
-	"github.com/suborbital/reactr/rwasm/moduleref"
-	"github.com/suborbital/reactr/rwasm/runtime"
+	"github.com/suborbital/reactr/scheduler"
 
 	"github.com/pkg/errors"
 )
 
-//Runner represents a wasm-based runnable
-type Runner struct {
+//wasmRunner represents a wasm-based runnable
+type wasmRunner struct {
 	env *runtime.WasmEnvironment
 }
 
-// NewRunner returns a new *Runner
-func NewRunner(filepath string) *Runner {
+// newRunnerFromFile returns a new *wasmRunner
+func newRunnerFromFile(filepath string, api api.HostAPI) *wasmRunner {
```

Comment:
Since we are breaking compatibility anyway what are your thoughts on taking an io.ReadCloser instead of a file. This would allow us to support reading from a bytebuffer, http request body, etc without actually changing code.

Reply:
Great idea, will do

Reply:
Done